### DISCUSSION TITLE
Converted Timeline has different Extension from Newly Created

### DISCUSSION THREAD

**Discussion by mechPenSketch** [dialogic-godot/dialogic#1508] [Q&A] [answered]
When I convert my timeline from Dialogic 1.x, it ends in `.cnv`. But when I create a new timeline, it ends in `.dtl`. Is `.dtl` supposed to be the only extension for timelines, or is `.cnv` is intended to be an acceptable alternative?

**Comment by coppolaemilio** [comment, marked as answer]
`.dtl` is the only extension for timelines. `.cnv` is the conversion file.

**Comment by mechPenSketch** [comment]
Thanks for the clarification. With this in mind, I've understood the conversion better and made somes changes in this PR: https://github.com/coppolaemilio/dialogic/pull/1509
